packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -109,7 +110,8 @@ export default function MemberListItem(props: Props) {
       (props.member.role !== MembershipRole.OWNER ||
         ownersInTeam() > 1 ||
         props.member.id !== currentUserId)) ||
-    (props.team.membership?.role === MembershipRole.ADMIN && props.member.role !== MembershipRole.OWNER);
+    (props.team.membership?.role === MembershipRole.ADMIN && props.member.role !== MembershipRole.OWNER) ||
```

Comment:
We should do this for 'Other Teams' too (`organizations/pages/components/MemberListItem.tsx`)

Reply:
hi @CarinaWolli
![Screenshot from 2023-11-01 17-05-23](https://github.com/calcom/cal.com/assets/32706411/558ce24f-61b2-45a3-805c-4214b5a8a4a9)
the other-teams tab is only visible to the organization admin and owners right?


and they should have all acceess (like deleting, editing) and (resending invitation for pending members) 
![Screenshot from 2023-11-01 17-07-19](https://github.com/calcom/cal.com/assets/32706411/d71957cb-125a-4e45-855c-2bfbb5ec327d)
![Screenshot from 2023-11-01 17-07-04](https://github.com/calcom/cal.com/assets/32706411/1b873f24-ded8-40c1-ac37-9c1b6e2f9dd7)
then why are we using a seperate component for other team why not just use /teams/pages/team-members-view

Reply:
check this pr out #12196